src/Umbraco.Web.UI.Client/src/common/directives/components/media/umbmedianodeinfo.directive.js

```diff
@@ -10,53 +10,19 @@
                 scope.allowOpenMediaType = true;
 
                 // get document type details
-                scope.mediaType = getMediaType(scope.node);
+                scope.mediaType = scope.node.documentType.value;
```

Comment:
for media this is a `mediaType` so for consistency we could just call both `contentType` instead of having 2 different things.